src/components/plots/abstractPlot.ts

```diff
@@ -16,6 +16,12 @@ export module Plot {
     datasetKey: string
   }
 
+  export type PlotData = {
+    data: any[];
+    pixelPoints: Point[];
```

Comment:
We should query QE on this one.  I chose `pixelPoints` over `pixelPositions` since it highlights the fact that points are returned.